start_sshd.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -ex
```

Comment:
See various bash-ism notes from the first file.

Reply:
Done.